### PR TITLE
Branch list employee leave

### PR DESCRIPTION
**Pull request by pkwangjoo**
- Support ListLeavesByEmployeeCommand
- format: `llbe eid/EMPLOYEE_ID`

### CODE REVIEW

File: src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public static final String COMMAND_WORD = "llbe";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists the leaves taken by the specified employee"
+            + "Parameters: EMPLOYEE_ID \n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_EMPLOYEE + "101";
+
+    public static final String MESSAGE_SUCCESS = "Listed all leaves by employee %1$s";
+
+    public static final String MESSAGE_EMPLOYEE_NOT_FOUND = "No employee exists with the given employee id.";
```
Comment (jerrrren) [inline src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java:26]: Not your issue but I just recalled that we should be moving this message to commons

File: src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    private final Id employeeId;
+
+    /**
+     * Creates a ListLeavesByEmployee to list all leaves by the
```
Comment (jerrrren) [inline src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java:31]: Not sure if its a typo but u may have wanted to write *ListLeavesByEmployeeCommand

**Review by jerrrren — APPROVED** [review]
Generally LGTM

File: src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java (hunk 2 above)
Comment (pkwangjoo) [reply src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java:31]: okie thanks!

File: src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java (hunk 1 above)
Comment (pkwangjoo) [reply src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java:26]: yep

**Pull request merged**